proxy/api/ts/ts.h

```diff
@@ -2419,6 +2419,16 @@ tsapi const TSUuid TSProcessUuidGet(void);
 */
 tsapi const char *TSHttpTxnPluginTagGet(TSHttpTxn txnp);
 
+/*
+ * Return information about the client protocols
+ */
+tsapi TSReturnCode TSHttpTxnClientProtocolStackGet(TSHttpTxn txnp, int n, char const **result, int *actual);
+tsapi TSReturnCode TSHttpSsnClientProtocolStackGet(TSHttpSsn ssnp, int n, char const **result, int *actual);
+tsapi int TSHttpTxnClientProtocolStackContains(TSHttpTxn txnp, char const *tag, char const **ret_tag);
+tsapi int TSHttpSsnClientProtocolStackContains(TSHttpSsn ssnp, char const *tag, char const **ret_tag);
+tsapi char const *TSNormalizedProtocolTag(char const *tag);
+tsapi char const *TSRegisterProtocolTag(char const *tag);
```

Comment:
Can we please be consistent about ``const char`` and ``char const``? Let's choose one or the other and file a separate Jira to make the API consistent.